addons/stock/models/stock_orderpoint.py

```diff
@@ -212,7 +214,28 @@ def action_stock_replenishment_info(self):
         action['res_id'] = res.id
         return action
 
+    @api.depends('product_id', 'qty_to_order', 'product_max_qty')
+    def _compute_unwanted_replenish(self):
+        for orderpoint in self:
+            orderpoint.unwanted_replenish = (orderpoint.product_id.virtual_available + orderpoint.qty_to_order) > orderpoint.product_max_qty
+
     def action_replenish(self):
+        # prevent Reordering too much
+        if not(self.env.context.get('force_unwanted_replenish', False)):
+            infinite_forecast = any(orderpoint.unwanted_replenish for orderpoint in self)
+            ctx = dict(self.env.context or {})
+            if infinite_forecast:
+                ctx.setdefault('active_ids', [orderpoint.id for orderpoint in self])
```

Comment:
You can use default_stock_orderpoint_ids as context. Would allow you to remove the default in the wizard